src/array_api_stubs/_draft/elementwise_functions.py

```diff
@@ -514,7 +518,7 @@ def atan(x: array, /) -> array:
     """
 
 
-def atan2(x1: array, x2: array, /) -> array:
+def atan2(x1: Union[array, float], x2: Union[array, float], /) -> array:
```

Comment:
Same here as my `hypot` comment https://github.com/data-apis/array-api/pull/862/files#r1857364664

Reply:
Same response. If we allow `int`, how that gets converted to a float is undefined. Hence, the types stay true to the parameter descriptions where we specify that `x1` and `x2` should have real-valued floating-point data types.